src/app/login/login.component.scss

```
@@ -0,0 +1,50 @@
+$white: #ffffff;
```

pensándolo mejor... porqué no creo un archivo en `src/app/styles` (por ejemplo) llamado `_variables.scss` en el que meter las variables generales de las hojas de estilo :thinking: 
Y aquí haría 
```suggestion
@use '/app/styles/variables'
```
(no estoy seguro de si la ruta es así o relativa)
Debo eliminar el resto de las variables declaradas